app.py

```diff
@@ -129,24 +171,214 @@ async def command(ack, body, respond, client, logger):
+            },
+            "label": {
+                "type": "plain_text",
+                "text": "The PAX",
+                "emoji": True
+            }
+        },
+        {
+            "type": "input",
+            "block_id": "fngs",
+            "element": {
+                "type": "plain_text_input",
+                "action_id": "fng-action",
+                "initial_value": "None",
+                "placeholder": {
+                    "type": "plain_text",
+                    "text": "FNGs"
+                }
+            },
+            "label": {
+                "type": "plain_text",
+                "text": "List untaggable names separated by commas (FNGs, Willy Lomans, etc.)"
+            }
+        },
+        {
+            "type": "input",
+            "block_id": "count",
+            "element": {
+                "type": "plain_text_input",
+                "action_id": "count-action",
+                "placeholder": {
+                    "type": "plain_text",
+                    "text": "Total PAX count including FNGs"
+                }
+            },
+            "label": {
+                "type": "plain_text",
+                "text": "Count"
+            }
+        },
+        {
+            "type": "input",
+            "block_id": "moleskine",
+            "element": {
+                "type": "plain_text_input",
+                "multiline": True,
+                "action_id": "plain_text_input-action",
+                "initial_value": "WARMUP: \nTHE THANG: \nMARY: \nANNOUNCEMENTS: \nCOT: ",
+                "placeholder": {
+                    "type": "plain_text",
+                    "text": "Tell us what happened\n\n"
+                }
+            },
+            "label": {
+                "type": "plain_text",
+                "text": "The Moleskine",
+                "emoji": True
+            }
+        },
+        {
+            "type": "divider"
+        },
+        {
+            "type": "section",
+            "block_id": "destination",
+            "text": {
+                "type": "plain_text",
+                "text": "Choose where to post this"
+            },
+            "accessory": {
+                "action_id": "destination-action",
+                "type": "static_select",
+            "placeholder": {
+                "type": "plain_text",
+                "text": "Choose where"
+            },
+            "initial_option": initial_channel_option,
+            "options": channel_options
+            }
+        }
+    ]
+
+    if config('EMAIL_TO', default=''):
+        blocks.append({
+            "type": "input",
+            "block_id": "email",
+            "element": {
+                "type": "plain_text_input",
+                "action_id": "email-action",
+                "initial_value": config('EMAIL_TO', default='')
+            },
+            "label": {
+                "type": "plain_text",
+                "text": "Email"
+            }
+        })
```

Comment:
Add to break up the building of the slack blocks to conditionally add the email block.

Reply:
so this emails a copy to the address specified?

Reply:
@wolfpackt99 correct. If GMAIL_USER config exists, then the email input block appears in the modal. Then if that input field, email_to has a value, then the post is emailed.